pages/admin/lessons/[lessonSlug]/[pageName]/index.tsx

```diff
@@ -39,26 +53,76 @@ type Module = {
 }
 type Modules = Module[]
 
-type ContentProps = {
-  pageName?: string | string[]
+const IntroductionPage = ({ lesson }: { lesson: Lesson }) => {
+  const [updateLesson] = useUpdateLessonMutation()
+
+  const [formOptions, setFormOptions] = useState(
+    getPropertyArr(lesson, ['challenges', '__typename'])
+  )
+
+  // Update the Inputs values when the lesson changes
```

Comment:
I think you forgot to remove these commented out lines

Reply:
Oops, yeah.